eight-week-challenge/eight-week-challenge/HomeViewController.m

```diff
@@ -8,6 +8,7 @@
 
 #import "HomeViewController.h"
 #import "LeaderboardCell.h"
+#import "StartDate.h"
```

Comment:
Don't worry about this now, it's low-priority, but this is not a great name for the class -- we're not modeling a start date, we just want a method that gives us the program week and day numbers based on the length of time since the program's start date.

This was the example method invocation:

```
[ProgramDay currentWeekAndDayFromStartDate: startDate todaysDate: todaysDate];
```